index.html

```diff
@@ -102,9 +96,39 @@ <h2 class="face">Facebook 360</h2>
             <li>css</li>
             <li>javascript</li>
           </ul>
-          <button class="btn">See project</button>
+          <button class="btn" id="myBtn1">See project</button>
         </div>
       </section>
+      <div id="myModal" class="modal">
+        <div class="modal-container">
+          <div class="modal-content">
+            <div class="modal-title">
+              <div class="dflex">
+                <h1 id="modalTitle">Title</h1>
+
+                <span class="xp">&times;</span>
+                </div>
+
+              <div id="modalSubt" class="modal-nav black"></div>
+                <img id="modalImg" class="modal-img" src="img/Portfolio1-big.svg" alt="Tonic">
+
+            </div>
+            <div class="modal-text toggle-body">
+            
+              <p id="modalDesc" class="toggle-p"></p>
+              
+            
+              <div class="modal-footer">
+                <ul id="modalTech" class="toggle-list-top "></ul>
+                <div class="btn-modal">
+                <button id="modalBtn1" type="button" class="btn-toggle">See Live &nbsp; <img class="icon" src="/img/liveIcon.svg" alt="Live"></button>
+                <button id="modalBtn2" type="button" class="btn-toggle">See Source &nbsp; <img class="icon" src="/img/sourceIcon.svg" alt="Source"></button>
```

Comment:
- [ ] These icons in these modal buttons are not displaying correctly. Please fix it so that your work looks better 😎 
![image](https://user-images.githubusercontent.com/46839548/162578490-ba99a327-5d5f-44ef-9241-30eebe6e18e3.png)